automate/scripts/etl_utils_test.py

```diff
@@ -0,0 +1,553 @@
+import unittest
+import docker
+from unittest.mock import patch, MagicMock, mock_open
+from io import StringIO
+from etl_utils import unzip_vocab,calculate_retry_delay,retrieve_load_status,update_json_file,read_file,download_hash_from_s3,read_version_and_md5_hash_from_json,remove_folder,remove_docker_volume,truncate_omop_tables,connect_to_database,download_from_s3,download_latest_vocab_from_s3,run_docker_compose, run_docker_compose_with_logs, remove_docker_containers,delete_files_from_given_path
+import subprocess
+import os
+import coverage
+import psycopg2
+import json
+
+
+
+class TestRunDockerCompose(unittest.TestCase):
+    @patch('builtins.print')  # Mock the print function
+    def test_run_docker_compose(self, mock_print):
+        docker_client = docker.from_env()
+        yaml_path = "docker-compose-hello-world.yaml"
+
+        # Run the function
+        run_docker_compose(yaml_path)
+
+        # Check if the containers are running
+        containers = docker_client.containers.list()
+        self.assertTrue(containers)  # Assert that there are containers running
+        for container in containers:
+            print(f"container status {container.status}")
+            self.assertEqual(container.status, "exited")  # Assert that each container is running
```

Comment:
nit: I think printing status is not necessary in unit test. Assertion is enough.